homework_03/17_petar_damyanov/test_circle.py

```diff
@@ -0,0 +1,38 @@
+from circle import Circle, Point, RelativePosition
+
+def test_no_common_points():
```

Comment:
It's good you've done multiple tests for a single case, it ensures the code works with various parameters